paddle/operators/name_convention.md

```diff
@@ -0,0 +1,15 @@
+## Operator Name Convention
+
+To make the operator document itself more clear. we recommend operator names observe the listing conventions.
+
+### Input/Output names
+
+* Variable name is prefer uppercase. e.g. `X`, `Y`. But when the variable is tensor, its name should lowercase. e.g. `matrix`, to discriminate with otherone.
+
+* element wise operator, math operator or similar op, please obey common name convention. if the operator only have one output, use `Out`.
```

Comment:
I think we shall distinguish between input/outputs' `ProtoName` and `DescName`.

Reply:
Done.